src/Audio/CueData.cs

```diff
@@ -663,36 +682,98 @@ public XACTClip(BinaryReader reader, double clipVolume, byte filterType)
 						weights
 					);
 				}
-				else if (eventType == 7) // SetPitchEvent
```

Comment:
Yeah, let's do the enum. At this point I'm 99% sure it's just a long-ass enum that feature-creeped as XACT got further into development. The recurrence enums are probably just enums copypasta'd after the pre-existing enums to add the recurrence feature.